core-bundle/src/Twig/Loader/ContaoFilesystemLoader.php

```diff
@@ -105,16 +108,35 @@ public function getSourceContext(string $name): Source
             return new Source(file_get_contents($path), $templateName, $path);
         }
 
+        $getExtendedTemplate = static function ($path): string|null {
+            if (
+                1 === preg_match(
+                    '/\$this\s*->\s*extend\s*\(\s*[\'"]([a-z0-9_-]+)[\'"]\s*\)/i',
+                    (string) file_get_contents($path),
```

Comment:
Why the string cast? Is it, because `$path` could be invalid coming from `TemplateLoader#getDefaultPath()`?

Reply:
The string cast was already there. I think this is probably something PHPStan wants.